geode-core/src/main/java/org/apache/geode/cache/client/PoolFactory.java

```diff
@@ -136,14 +136,6 @@
    */
   int DEFAULT_STATISTIC_INTERVAL = -1;
 
-  /**
-   * The default value for whether connections should have affinity to the thread that last used
-   * them.
-   * <p>
-   * Current value: <code>false</code>.
-   */
-  boolean DEFAULT_THREAD_LOCAL_CONNECTIONS = false;
```

Comment:
Should this be removed or deprecated?

Reply:
Since this is part of the public API it should be deprecated

Reply:
It’s package level protected. I think it can be removed though. I can’t recall why I had to leave it. I’ll check again.

Reply:
Sorry, misread what y'all were saying. I removed it because it isn't used internally anymore and it was package private, which isn't public API.

Reply:
Sorry again, I'm tired, yes It is part of the API, PoolFactory is an Interface. Fixing now.